lix/packages/server/src/util/getEnv.ts

```diff
@@ -0,0 +1,22 @@
+import { config } from "dotenv"
+
+// we configure the dotenv with default - which loads the .env file from the execution folder
+config()
+
+export function getEnvVar(varName: string, descirption: string, defaultValue?: string) {
```

Comment:
better have call signature getEnvVar(varname, {default, description})   its more expandable and clear what the args are.